<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hangman</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QPushButton" name="buttonEnter">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputChar">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>1</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelAttempted">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Attempted</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="attemptedChars">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonResign">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Give me the answer!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonNewWord">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>New word!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonHint">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Give me a hint!</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>421</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelNumAttempts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Attempts: 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAttemptsLeft">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Attempts Left:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChoose_Word_List">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionChoose_New_Word_List"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuChoose_Word_List"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionChoose_New_Word_List">
   <property name="text">
    <string>Choose New Word List</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
